src/Symfony/Component/Security/Csrf/CsrfTokenManagerInterface.php

```diff
@@ -30,7 +30,7 @@ interface CsrfTokenManagerInterface
      *
      * @return CsrfToken The CSRF token
      */
-    public function getToken($tokenId);
+    public function getToken(string $tokenId): string;
```

Comment:
Adding a return type will break current implementations of the interface. Please concentrate on parameter type-hints for now.